ktor-http/common/src/io/ktor/http/Parameters.kt

```diff
@@ -10,6 +10,12 @@ import io.ktor.util.*
  * Represents HTTP parameters as a map from case-insensitive names to collection of [String] values
  */
 interface Parameters : StringValues {
+    /**
+     * Returns a [UrlEncodingOption] instance
+     */
+    @KtorExperimentalAPI
+    fun urlEncodingOption(): UrlEncodingOption = UrlEncodingOption.DEFAULT
```

Comment:
Shouldn't it be a property instead of function?

Reply:
I thought that adding a variable to this interface would break compatibility since all implementation classes should override the variable.

Reply:
You can define default values for properties too:
```kotlin
val urlEncodingOption: UrlEncodingOption get() = UrlEncodingOption.DEFAULT
```